code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -170,7 +170,7 @@
 	desc = "A special helmet designed for work in a hazardous, low-pressure environment. Has radiation shielding."
 	icon_state = "hardsuit0-engineering"
 	item_state = "eng_helm"
-	armor = list("melee" = 30, "bullet" = 5, "laser" = 10, "energy" = 12, "bomb" = 10, "bio" = 100, "rad" = 75, "fire" = 100, "acid" = 75)
+	armor = list("melee" = 30, "bullet" = 5, "laser" = 10, "energy" = 12, "bomb" = 10, "bio" = 100, "rad" = 100, "fire" = 100, "acid" = 75)
```

Comment:
This would make Engineering Hardsuits a straight up upgrade from the normal radiation suits. Not really a fan of that.

Reply:
In my opinion the engi hardsuit is supposed to be a radsuit alternative not a lesser option that can go into space.